index.bs

```diff
@@ -1043,6 +1045,608 @@ Note: Invocations of {{MLContext/compute()}} will fail if any of the {{MLContext
+  MLConv2dSupportLimits conv2d;
+  MLConv2dSupportLimits convTranspose2d;
+  MLBinarySupportLimits add;
+  MLBinarySupportLimits sub;
+  MLBinarySupportLimits mul;
+  MLBinarySupportLimits div;
+  MLBinarySupportLimits max;
+  MLBinarySupportLimits min;
+  MLBinarySupportLimits pow;
+  MLBinarySupportLimits equal;
+  MLBinarySupportLimits greater;
+  MLBinarySupportLimits greaterOrEqual;
+  MLBinarySupportLimits lesser;
+  MLBinarySupportLimits lesserOrEqual;
+  MLLogicalNotSupportLimits logicalNot;
+  MLSingleInputSupportLimits abs;
+  MLSingleInputSupportLimits ceil;
+  MLSingleInputSupportLimits cos;
+  MLSingleInputSupportLimits erf;
+  MLSingleInputSupportLimits exp;
+  MLSingleInputSupportLimits floor;
+  MLSingleInputSupportLimits identity;
+  MLSingleInputSupportLimits log;
+  MLSingleInputSupportLimits neg;
+  MLSingleInputSupportLimits reciprocal;
+  MLSingleInputSupportLimits sin;
+  MLSingleInputSupportLimits sqrt;
+  MLSingleInputSupportLimits tan;
+  MLSingleInputSupportLimits elu;
+  MLSingleInputSupportLimits expand;
+  MLGatherSupportLimits gather;
+  MLSingleInputSupportLimits gelu;
+  MLGemmSupportLimits gemm;
+  MLGruSupportLimits gru;
+  MLGruCellSupportLimits gruCell;
+  MLSingleInputSupportLimits hardSigmoid;
+  MLSingleInputSupportLimits hardSwish;
+  MLNormalizationSupportLimits instanceNormalization;
+  MLNormalizationSupportLimits layerNormalization;
+  MLSingleInputSupportLimits leakyRelu;
+  MLSingleInputSupportLimits linear;
+  MLLstmSupportLimits lstm;
+  MLLstmCellSupportLimits lstmCell;
+  MLBinarySupportLimits matmul;
+  MLSingleInputSupportLimits pad;
+  MLSingleInputSupportLimits averagePool2d;
+  MLSingleInputSupportLimits l2Pool2d;
+  MLSingleInputSupportLimits maxPool2d;
+  MLPreluSupportLimits prelu;
+  MLSingleInputSupportLimits reduceL1;
+  MLSingleInputSupportLimits reduceL2;
+  MLSingleInputSupportLimits reduceLogSum;
+  MLSingleInputSupportLimits reduceLogSumExp;
+  MLSingleInputSupportLimits reduceMax;
+  MLSingleInputSupportLimits reduceMean;
+  MLSingleInputSupportLimits reduceMin;
+  MLSingleInputSupportLimits reduceProduct;
+  MLSingleInputSupportLimits reduceSum;
+  MLSingleInputSupportLimits reduceSumSquare;
+  MLSingleInputSupportLimits relu;
+  MLSingleInputSupportLimits resample2d;
+  MLSingleInputSupportLimits reshape;
+  MLSingleInputSupportLimits sigmoid;
+  MLSingleInputSupportLimits slice;
+  MLSingleInputSupportLimits softmax;
+  MLSingleInputSupportLimits softplus;
+  MLSingleInputSupportLimits softsign;
+  MLSingleInputSupportLimits split;
+  MLSingleInputSupportLimits tanh;
+  MLSingleInputSupportLimits transpose;
+  MLSingleInputSupportLimits triangular;
+  MLWhereSupportLimits where;
+};
+</script>
+
+<dl dfn-type=dict-member dfn-for=MLOpSupportLimits>
+    : <dfn>preferredInputLayout</dfn>
+    :: Preferred input layout for layout aware operators like {{MLGraphBuilder/conv2d(input, filter, options)}}.
+    : <dfn>input</dfn>
+    :: Support limits for operator input.
```

Comment:
Each of these should link to the operator, such as [`input()`](https://webmachinelearning.github.io/webnn/#api-mlgraphbuilder-input).